data/signersDefault.json

```diff
@@ -325,6 +325,12 @@
 			"sdid": "buyon.com",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL"
+		}, {
+			"domain": "cacf-festgeld.de",
+			"addr": "*@service.cacf-festgeld.de",
+			"sdid": "service.cacf-festgeld.de",
```

Comment:
You can leave it as is, but typically I add the parent domain (i.e. `cacf-festgeld.de`) too, just in case they start using that for signing in the future.

Reply:
At the moment, mails from service.cacf-festgeld.de are signed with SDID service.cacf-festgeld.de. Mails from cacf-festgeld.de aren't signed at all.
I  guess I will have to change the rule type, if I add the base domain. 
Otherwise, getting an unsigned mail from `...@cacf-festgeld.de` will produce a signing error?

Reply:
If you already know that mails by `...@cacf-festgeld.de` are not signed, restricting this rule is correct.

Most of the time I additionally add a neutral rule for the top domain to make it explizit, but this should not be needed.